Tella/Utils/Extensions/URLExtension.swift

```diff
@@ -196,21 +217,28 @@ extension URL {
         return self.deletingLastPathComponent()
     }
     
-    
-    nonisolated func trimAudio(newName: String,
+    nonisolated func trimMedia(newName: String,
                                startTime: Double,
-                               endTime: Double) async throws -> URL {
+                               endTime: Double,
+                               type: FileExtension) async throws -> URL {
         
         let asset = AVAsset(url: self)
         let startTime = CMTime(seconds: startTime, preferredTimescale: 600)
         let endTime = CMTime(seconds: endTime, preferredTimescale: 600)
         let duration = CMTimeSubtract(endTime, startTime)
         
-        let exportSession = AVAssetExportSession(asset: asset, presetName: AVAssetExportPresetAppleM4A)
+        let exportSession = AVAssetExportSession(asset: asset, presetName: AVAssetExportPresetHighestQuality)
         
         let outputURL = createURL(name: newName)
         exportSession?.outputURL = outputURL
-        exportSession?.outputFileType = .m4a
+        
+        switch type {
+        case .mov:
+            exportSession?.outputFileType = AVFileType.mov
```

Comment:
I think it's better to use `getAVFileType` function to get the `AVFileType` instead of using `type`